CONTRIBUTING.md

```diff
@@ -89,8 +94,10 @@ pip install -r common_build/taskipy/requirements.txt -c common_build/taskipy/con
 
 You can then use `task` to run various common tasks:
 
-- `task tests` to run all the tests (including type checking);
+- `task tests` to run all the normal tests (including type checking);
```

Comment:
I am no longer sure what to expect here from the description. This command runs all of the unit and integration tests, along with the type checking, but _not_ some tests which are marked as slow or linting? Perhaps this could say something like `run all tests (including type checking), but excluding slow tests`?